components/kyma-environment-broker/internal/orchestration/handlers/handlers.go

```diff
@@ -51,10 +51,5 @@ func defaultOrchestrationStrategy(spec *orchestration.StrategySpec) {
 		spec.Type = orchestration.ParallelStrategy
 	}
 
-	switch spec.Schedule {
-	case orchestration.MaintenanceWindow:
-	case orchestration.Immediate:
-	default:
-		spec.Schedule = orchestration.Immediate
-	}
+	spec.Schedule = string(orchestration.Immediate)
```

Comment:
I would remove the `defaultOrchestrationStrategy` function, and force client to provide the immediate/now or date with the validation functions you implemented in kyma handler (which would be moved here)